app/src/main/java/com/sopt/now/compose/ui/follower/FollowerScreen.kt

```diff
@@ -0,0 +1,55 @@
+package com.sopt.now.compose.ui.follower
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Card
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import coil.compose.AsyncImage
+import com.sopt.now.compose.network.reponse.ResponseFollowerDto
+
+@Composable
+fun FollowerScreen() {
+    val viewModel: FollowerViewModel = viewModel()
+    val followers = viewModel.followers.collectAsState().value
```

Comment:
```suggestion
    val followers by viewModel.followers.collectAsStateWithLifecycle()
```

- by 사용하는게 가독성에 더 좋을 것 같습니다.
- collectAsState와 collectAsStateWithLifecycle의 차이는 뭘까요?

Reply:
오..!
collectAsStateWithLifecycle 얘는 Composable 생명주기에 따라서 작동해서 포그라운드일 때만 state 업데이트를 해주고, collectAsState는 포그라운드가 아니여도 state가 계속 업데이트 됩니다!